src/components/styles/BookForm.css

```diff
@@ -1,36 +1,66 @@
-/* BookForm container */
 .book-form {
+  width: 100%;
   display: flex;
-  flex-direction: column;
-  max-width: 400px;
-  margin: 0 auto;
+  justify-content: center;
+  padding: 20px;
+  margin-bottom: 20px;
 }
 
-/* Form input fields */
-.book-form div {
-  margin-bottom: 1rem;
+.form-row {
+  display: flex;
+  align-items: center;
+  gap: 10px;
+  flex-wrap: wrap;
 }
 
-.book-form input {
-  width: 100%;
-  padding: 0.5rem;
-  border: 1px solid #ccc;
-  border-radius: 4px;
-  font-size: 1rem;
+input[type="text"],
+.form-select,
+button {
+  padding: 12px;
+  border: none;
+  border-radius: 5px;
+  flex: 1;
+}
+
+input[type="text"],
+.form-select {
+  min-width: 350px;
+}
+
+.form-select {
+  background-color: #fff;
+  cursor: pointer;
+  transition: border-color 0.3s ease-in-out, background-color 0.3s ease-in-out;
+}
+
+.form-select:focus {
+  outline: none;
+  background-color: #fff;
 }
 
-/* Submit button */
 .book-form button {
-  background-color: #3498db;
-  color: white;
-  border: none;
-  padding: 0.5rem 1rem;
-  border-radius: 4px;
-  font-size: 1rem;
+  background-color: #007bff;
+  color: #fff;
   cursor: pointer;
-  transition: background-color 0.3s ease;
+  min-width: 200px;
+  transition: background-color 0.3s ease-in-out;
+  margin-right: 40px;
+}
```

Comment:
- [ ] It's wonderful🎉😍 that you've made an effort to adhere to the previous reviewer's [guidelines](https://github.com/Xghayor/Book-store/pull/5/files#r1312526627). 
- [ ] However, the 'Add Book' button doesn't😭 quite align with the Zeplin design. 
- [ ] Could you please🙏🙏 ensure that you use percentage units instead of fixed pixel units? 

- [ ] I believe this adjustment will help your design meet the project requirements and stand out to potential recruiters in the future.🤔🤔
- [ ] Your design:
![screencapture-localhost-3000-2023-09-02-00_34_11](https://github.com/Xghayor/Book-store/assets/118761303/36e8acc6-64d1-452a-b9a2-1430805e9307)